### PR TITLE
Added support for more primitives

### PR DESCRIPTION
**Pull request by euforia**
Added support for all uints and ints.  This was validated against a hashicorp nomad job spec.

### CODE REVIEW

File: nodes_test.go [hunk 1]
```diff
@@ -426,13 +429,13 @@ func TestTokenize(t *testing.T) {
...
-			token.Token{},
-			true,
-		},
+		// {
+		// 	"unsupported",
+		// 	reflect.ValueOf(uint8(1)),
+		// 	false,
+		// 	token.Token{},
+		// 	true,
+		// },
```
Comment (rodaine) [inline nodes_test.go]: please just delete this test

**Review by rodaine — COMMENTED** [review]
Awesome, thanks for this! Just the one minor comment.

File: nodes_test.go (hunk 1 above)
Comment (euforia) [reply nodes_test.go]: Whoops. Should be good now.

**Review by rodaine — APPROVED** [review]
Thanks!

**Pull request merged**